scripts/downloaded_validation_script.sh

```diff
@@ -142,7 +142,7 @@ runHapiValidator() {
   validatorversion_underscore=${validatorversion//./_}
   validatordestination=~/.fhir/validators/validator_cli_v$validatorversion_underscore.jar
   # Concatenate folders_for_validation in fhir directory
-  folders_to_validate=""
+  folders_to_validate=" -ig $fhir_folder_path"
   for package in $( (ls -d $fhir_folder_path/*/package)); do
```

Comment:
I would expect to find the external dependencies as well in this folder structure. 
Example:
![image](https://user-images.githubusercontent.com/10847488/204321180-31390627-b929-4520-b936-5f85ee5a8734.png)

Reply:
It is because the 'package' folder is missing when copied with https://github.com/gematik/spec-TemplateForSimplifierProjects/blob/936bb7835743506b96b4a6c1aa40e4d716e8d8b7/scripts/downloaded_validation_script.sh#L135 ?

Reply:
KBV Schlüsseltabellen in .fhir-packages are not under versioning-control, makes collaborative work harder

Reply:
Thats why they are in the ExternalDependencies folder of the projects and are copied at each run of the script.